Transformer.md

```diff
@@ -1,31 +1,30 @@
 # Transformers
 
-Transformer is used to turn a dataset from a type `Event` ([Events](Events.md)) 
-into another of the same type and another subtype. A  `Transformer` is a class and can accept as a parameter 
+A transformer is used to turn one or multiple  datasets  
+into another one. A  `Transformer`  accept a configuration class as a parameter 
 a configuration class of type
  [TransformerConfig](https://github.com/X-DataInitiative/SCALPEL-Extraction/blob/master/src/main/scala/fr/polytechnique/cmap/cnam/etl/transformers/TransformerConfig.scala).
-This configuration file is a factory to save values and pass them later to the different methods.
+This configuration file control the behaviour of the `Transformer` through its methods and values.
 
 
-`transform` is the main method of `Transformer`, this method accepts one or several datasets of type `Event` as input
- and after the different transformations it returns a dataset of type `Event` and a different subtype.
+`transform` is the main and the unique public available method of `Transformer`, this method accepts one or several datasets  as input
+ and after the applying the logic of the transformation, returns a dataset of type `Event`([Events](Events.md)) with a new `category`.
```

Comment:
It is either
`and after applying of transformation logic`
Or
`and after the application of the transformation logic`

I prefer the second phrasing.